distributions/standalone/assembly-allowlist.txt

```diff
@@ -120,8 +120,8 @@
 /mule-standalone-${productVersion}/lib/mule/mule-extensions-annotations-1.0.0-alpha-1.jar
 /mule-standalone-${productVersion}/lib/mule/mule-extensions-api-1.0.0-alpha-1.jar
 /mule-standalone-${productVersion}/lib/mule/mule-module-annotations-${productVersion}.jar
-/mule-standalone-${productVersion}/lib/mule/mule-module-apikit-3.9.6-SNAPSHOT.jar
-/mule-standalone-${productVersion}/lib/mule/mule-module-apikit-spi-3.9.6-SNAPSHOT.jar
+/mule-standalone-${productVersion}/lib/mule/mule-module-apikit-3.10.0-SNAPSHOT.jar
```

Comment:
Why is this version from apikit boost to 3.10? If there is a change needed probably thay have to cherry-pick it to their 3.9.x support branch.

Reply:
I have requested them doe updating that beanch